ironfish-cli/src/commands/fees.ts

```diff
@@ -49,9 +49,9 @@ export class FeeCommand extends IronfishCommand {
       'The fee rate for each transaction is computed by dividing the transaction fee in $ORE by the size of the transaction in kB.\n',
     )
     this.log('The low, medium, and high rates each come from a percentile in the distribution:')
-    this.log(`low:    ${low}th`)
-    this.log(`medium: ${medium}th`)
-    this.log(`high:   ${high}th`)
+    this.log(`slow:    ${slow}th`)
+    this.log(`average: ${average}th`)
+    this.log(`fast:   ${fast}th`)
```

Comment:
```suggestion
    this.log(`fast:    ${fast}th`)
```

nit - to align the percentiles